dtls_time.c

```diff
@@ -64,13 +69,13 @@ dtls_ticks(dtls_tick_t *t) {
   *t = k_uptime_get();
 }
 
-#elif defined(WITH_POSIX)
+#elif defined(WITH_POSIX) || defined(IS_WINDOWS)
 
 time_t dtls_clock_offset;
 
 void
 dtls_clock_init(void) {
-#ifdef HAVE_TIME_H
+#if defined(HAVE_TIME_H) || defined(_MSC_VER)
```

Comment:
Is there a reason for mixing `IS_WINDOWS` `_MSC_VER`?

At least the `HAVE_TIME_H`, why doesn't this work for CMake in Windows?

Reply:
I misread the macro here as `HAVE_SYS_TIME_H`. As `HAVE_TIME_H` is also available on Windows/Visual Studio, this change can actually be reverted :) (Done in ae898f80932a6df6e24fe54054948ec7ae816282.)